packages/cozy-client/src/helpers.js

```diff
@@ -40,7 +40,7 @@ const ensureFirstSlash = path => {
  * @param {string}   options.cozyUrl       Base URL of the cozy, eg. cozy.tools or test.mycozy.cloud
  * @param {Array}    options.searchParams  Array of search parameters as [key, value] arrays, eg. ['username', 'bob']
  * @param {string}   options.pathname      Path to a specific part of the app, eg. /public
- * @param {string}   options.hash          Path inside the app, eg. /files/test.jpg
+ * @param {string}   [options.hash]          Path inside the app, eg. /files/test.jpg
```

Comment:
Optional

Reply:
Maybe we should "standardize" our way to define optional params, because recently I used the ` Google Closure Compiler syntax` https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
The Google Closure Compiler syntax does not seem to be supported by typescript.

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

> . The parameter may also be declared optional by surrounding the name with square brackets:

```
/**
 * @param {string} [p4="test"] - An optional param with a default value
 */
```

Reply:
OK thanks! This is something to keep in mind when doing the same job for cozy-stack-client (for instance https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/SharingCollection.js#L61)